src/experiment/experiment.js

```diff
@@ -36,13 +37,9 @@ export const preloadObj2contentObj = (preloadObj) => {
   }, {});
 };
 
-// Automatically populate the audioContent object with the audio files
-// Copied audioContent from preload.js
-const audioBlocks = {
-  3: [mediaAssets.audio["feedbackCorrect"], mediaAssets.audio["feedbackIncorrect"]],
-};
-
-export const audioContent = preloadObj2contentObj(audioBlocks);
+const bucketURI = 'https://storage.googleapis.com/roar-mp' // TODO: separate buckets into "spanish" and "english" variants
```

Comment:
If we could move these out to something like experiment helpers that would be great.